packages/data-context/src/codegen/spec-options.ts

```diff
@@ -62,7 +62,7 @@ export class SpecOptions {
     const componentName = capitalize(camelCase(this.parsedPath.name))
     const frameworkOptions = {
       react: {
-        imports: ['import { mount } from "@cypress/react"', `import ${componentName} from "./${this.parsedPath.name}"`],
+        imports: ['import React from "react"', 'import { mount } from "@cypress/react"', `import ${componentName} from "./${this.parsedPath.name}"`],
```

Comment:
This change was made so that the generated component spec would actually run. The JSX in the spec would not load without this. Not sure if this was a miss or we just expect additional configuration?

Reply:
Does Vue need a similar import?

Reply:
Not sure either - but I can confirm Vue does not need this kind of import. So likely this was just missed.